src/ContractInteract/EIP20Gateway.js

```diff
@@ -535,8 +705,8 @@ class EIP20Gateway {
    */
   getLatestAnchorInfo() {
     return this.getAnchor().then((anchor) => {
-      anchor.getLatestStateRootBlockHeight().then((blockHeight) => {
-        anchor.getStateRoot(blockHeight).then((stateRoot) => {
+      return anchor.getLatestStateRootBlockHeight().then((blockHeight) => {
+        return anchor.getStateRoot(blockHeight).then((stateRoot) => {
```

Comment:
See https://github.com/OpenSTFoundation/mosaic.js/pull/56/files#diff-2046288c7009bc2988ab960e2a5cf646R599 you probably need to rebase.